test/distrib/gcf/python/README.md

```diff
@@ -0,0 +1,12 @@
+# Python Google Cloud Functions Distribtest
+
+This distribtest acts as a smoke test for usage of the `grpcio` Python wheel in
+the GCF environment. This test is dependent on two long-lived GCP resources:
+
+- `gcf-distribtest-topic` Pub Sub Topic
+- `grpc-gcf-distribtest` GCS Bucket. This bucket has 1 day TTL on all artifacts.
```

Comment:
optional: should we add a creation script or note of commands for their creation?

Reply:
I didn't actually use the CLI to create these -- I used the web UI. I've added a bit more detail to make sure all of the configuration is captured here.